edi_exchange_template_oca/models/edi_exchange_template_mixin.py

```diff
@@ -13,6 +13,19 @@
 _logger = logging.getLogger(__name__)
 
 
+def date_to_datetime(dt):
+    """Convert date to datetime."""
+    if isinstance(dt, datetime.date):
+        return datetime.datetime.combine(dt, datetime.datetime.min.time())
+    return dt
```

Comment:
Could it make sense to add timezone info?
Also, `datetime.datetime.min.time()` can be replaced by importing `time` from `datetime` and then using `time.min`.

Reply:
TZ I don't think is needed here. I don't see the point of replacing the min time var. What's the pro?